src/widgets/analytics/analytics.ts

```diff
@@ -241,6 +241,16 @@ const analytics: AnalyticsWidget = function analytics(widgetParams) {
         window.removeEventListener('beforeunload', onUnload);
       }
     },
+
+    getRenderState(renderState) {
+      return renderState;
```

Comment:
you need to add the local getWidgetRenderState call here :)

Reply:
you're right https://github.com/algolia/instantsearch.js/pull/4537/commits/949db21de0cdff9e843bfddb38b5c2d9c697a169